src/common/reducers/settings/reducers.js

```diff
@@ -85,6 +85,11 @@ function java(
       return { ...state, memory: action.memory };
     case ActionTypes.UPDATE_JAVA_PATH:
       return { ...state, path: action.path };
+    case ActionTypes.UPDATE_RESOLUTION:
+      return {
+        ...state,
+        resolution: action.resolution
+      };
```

Comment:
This is not directly related to java. We might want to create a new tab in settings called `Minecraft Settings`